hphp/runtime/base/runtime-option.h

```diff
@@ -513,6 +513,7 @@ struct RuntimeOption {
   F(uint32_t, JitMaxTranslations,      10)                              \
   F(uint32_t, JitMaxProfileTranslations, 30)                            \
   F(uint64_t, JitGlobalTranslationLimit, -1)                            \
+  F(uint64_t, JitMaxRequestTranslationTime,   -1)                       \
```

Comment:
This is an unsigned type.  The comparison in `tc-internal.cpp:130` will always succeed, and the compiler doesn't like that.  The setting above this does the same thing, except in that case -1 becomes effectively unlimited.  The same thing can be done here by removing the check for `maxTransTime >= 0` below.  I prefer making this a signed int as it is more clear.